packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -161,14 +194,16 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
 
             // Look for candidates used to generate the CSS
             context.scanner = new Scanner({ sources })
+            env.DEBUG && I.end('Setup scanner')
```

Comment:
wonder if we should reassign `env.DEBUG` to a local variable so we save on the repeated property access